cicd-stable.groovy

```diff
@@ -0,0 +1,14 @@
+node('linux')
+{
+  stage ('Poll') {
+    checkout([
+      $class: 'GitSCM',
+      branches: [[name: '*/main']],
+      doGenerateSubmoduleConfigurations: false,
+      extensions: [],
+      userRemoteConfigs: [[url: 'https://github.com/ZOSOpenTools/protobufport.git']]])
```

Comment:
We should also poll from upstream so that we trigger a build on their commits:
like this:
https://github.com/ZOSOpenTools/libiconvport/blob/main/cicd-dev.groovy#L5-L10